README.md

```diff
@@ -49,6 +49,21 @@ Launch the OpenFin runtime:
 npm run openfin
 ```
 
+### NPM Tasks
+
+Run webpack-dev-server with the development config: ``` npm run start ```
+
+Launch OpenFin with the development setup: ``` npm run openfin:dev ```
+
+Launch OpenFin with the production setup: ``` npm run openfin:prod ```
```

Comment:
Is it worth noting that to launch OpenFin with either of these configs, that a server must be running?

Could possibly group the related commands in the order which they'd usually be run, e.g.

---

#### Production build

Build and serve the production bundle:

```
npm run prod
```

Launch OpenFin with the production setup (requires the server to be running):

```
npm run openfin:prod
```